scripts/bcoin-init.js

```diff
@@ -18,14 +17,11 @@ let node;
     await logger.open();
     logger.info('LOGGER OPEN');
 
-    let prefix = process.env.prefix
-      ? process.env.prefix
-      : `${os.homedir()}/.bcoin/`;
-    const initScript = process.env.BCOIN_INIT_SCRIPT;
-
-    prefix = prefix.replace('~', os.homedir());
     // create the node with our custom configs
+    // the env option will pull options from the environment
+    // which is set in docker-compose.yml
```

Comment:
Do we want to explicitly say that env vars are being set in the `docker-compose.yml` because that could be changed or done another way